server/controllers/forums.js

```diff
@@ -0,0 +1,97 @@
+const Forum = require('../models/forums')
+const date = new Date()
```

Comment:
a new date must be created on every post otherwise forums will always show the server start date